runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -149,11 +149,11 @@ default boolean swapHarpoon()
 		position = 10,
 		keyName = "swapHome",

```

Comment:
What happens to existing configs with this? May be worth changing the key to something else..

Reply:
Good catch, it will throw exceptions until the value is updated. I'll fix this up real quick.

Reply:
Changed the keyName and ran it a few time, issue should be resolved.
Please let me know if you have any other issues/suggestions.